models/idea/index.js

```diff
@@ -352,6 +352,38 @@ class Idea extends Model {
     const cursor = await this.db.query(query, params);
     return await cursor.all();
   }
+
+
+  /**
+   * Read ideas with any of specified keywords in the title
+   * @param {string[]} keywords - list of keywords to search with
+   * @param {integer} offset - pagination offset
+   * @param {integer} limit - pagination limit
+   * @returns {Promise<Idea[]>} - list of found ideas
+   */
```

Comment:
More efficient search can be nice, actually. Especially if we get a lot of ideas and if the search has lower computational complexity (O(n) vs. O(1)?). It can be done later, can't it?

Reply:
opened issue #65 for this